docs/webiny-apps/page-builder/development/creating-iframe-element-plugin.md

```diff
@@ -0,0 +1,182 @@
+---
+id: creating-iframe-element-plugin
+title: Build a Plugin from scratch
+sidebar_label: Build a Plugin from scratch
+---
+
+This tutorial guides you through building a new element plugin for our Page Builder.
+Out of the box, we provide only the essential plugins. Writing your own plugin is not difficult, but you must be familiar with a couple of core concepts.
+
+> If you are not already familiar with how plugins work, we recommend
+> you first take a look at the [Plugins Crash Course](/docs/developer-tutorials/plugins-crash-course) before reading this article.
+
+Ready? Let's dive in!
+
+## What we'll build
+
+We’ll build an element plugin that will allow content creators to embed content using an iframe.
+
+Here's what the result will look like
+
+![Embed content with an iframe](/img/webiny-apps/page-builder/development/development/plugin-reference/editor/iframe/iframe-plugin.png)
+
+
+### Prerequisites
+
+This tutorial assumes you have already created a new Webiny project to work on. If not, we recommend going to our [quick start](/docs/get-started/quick-start.md) guide which will show you how to do it.
+
+### Need Help?
+
+Head over to our [Webiny forum](https://community.webiny.com/), [Gitter](https://gitter.im/Webiny/webiny-js)
+
+### Editor
+
+One of the main sections of the Page Builder App is the Editor, you can get there by editing a site that you create. By default, it offers you a variety of elements or plugins as seen in the image below.
+
+![Editor Elements](/img/webiny-apps/page-builder/development/development/plugin-reference/editor/iframe/editor-elements.png)
+
+
+When building pages, you are using these elements (e.g. Text, Image, Form, ...), which are registered in the system via plugins. In terms of rendering, every element has to have two plugins, one that defines how it's rendered in the actual page builder, and the other that defines how it's rendered on the website.
+
+If you are interested on different types of plugins, each app in Webiny consistes of the `types.ts` that includes all the plugin types of that app.
+In our case is [app-page-builder/src/types.ts](https://github.com/webiny/webiny-js/blob/master/packages/app-page-builder/src/types.ts)
+
+Let's render our plugin in the page builder, first we will use the `PbEditorPageElementPlugin` plugin type
```

Comment:
I would add two subtitles here. One "Editor Pugin", and the other "Render Plugin". Just so the user gets the distinction feeling even more.